packages/web3-validator/src/validation/string.ts

```diff
@@ -23,12 +23,12 @@ import { ValidInputTypes } from '../types';
 export const isString = (value: ValidInputTypes) => typeof value === 'string';
 
 export const isHexStrict = (hex: ValidInputTypes) =>
-	typeof hex === 'string' && /^(-)?0x[0-9a-f]*$/i.test(hex);
+	typeof hex === 'string' && /^((-)?0x[0-9a-f]+|(0x))$/i.test(hex);
```

Comment:
Maybe we need to check for capital X too?

Reply:
Thanks @nikoulai, I just added `0X` to the valid test cases.